build/validate.sh

```diff
@@ -1,14 +1,17 @@
 #!/bin/bash
-echo -e "\nChecking go fmt bosun.org/..."
-GOFMTOUT=$(go fmt -s bosun.org/...);
-if [ "$GOFMTOUT" != '' ]; then 
-    echo "The following files need gofmt -s:";
-    echo "$GOFMTOUT";
-    exit 1;
+echo -e "\nChecking gofmt -l -s -w for all folders that don't start with . or _"
+cd $GOPATH/src/bosun.org
+GOFMTRESULT=0
+GOFMTOUT=$(find . -maxdepth 1 -type d -iregex './[^._].*' -exec gofmt -l -s -w {} +);
```

Comment:
I'd say just always use find. Probably is a different flag we could use besides -exec, but there may be different regexes we want to use for other commands.

this is a pretty standard bash approach to running commands on specific files/folders taken from http://stackoverflow.com/a/13319190/17373